lib/pf/Switch/Fortinet/FortiGate.pm

```diff
@@ -269,7 +269,7 @@ sub returnAuthorizeVPN {
     if ( isenabled($self->{_VpnMap}) && $self->supportsVPNRoleBasedEnforcement()) {
         $logger->debug("Network device (".$self->{'_id'}.") supports roles. Evaluating role to be returned");
         if ( defined($args->{'user_role'}) && $args->{'user_role'} ne "" ) {
-            $role = $self->getRoleByName($args->{'user_role'});
+            $role = $self->getVpnByName($args->{'user_role'});
```

Comment:
Should we have a migration script to copy the role map to the VPN map?

Reply:
Looks like it does make sense to have one specific for the Fortigate since it is indeed a change of behavior

Reply:
Upgrade script uploaded